synapse/util/caches/expiringcache.py

```diff
@@ -140,6 +140,21 @@ def pop(self, key: KT, default: T = SENTINEL) -> Union[VT, T]:
 
         return value.value
 
+    def invalidate(self, keys: List[KT]) -> None:
```

Comment:
`invalidate` on the other caches just invalidates a single key. Let's do the same thing here to avoid confusion